src/pixelmapFeature.js

```diff
@@ -339,13 +339,14 @@ var pixelmapFeature = function (arg) {
       m_quadFeature = m_this.layer().createFeature('quad', {
         selectionAPI: false,
         gcs: m_this.gcs(),
-        visible: m_this.visible()
+        visible: m_this.visible(undefined, true)
```

Comment:
but that means that the user would have to provide answer to direct in their call, unless they are okay with passing undefined for direct in their code.